### PR TITLE
Add awesome-lite-websites

### PR DESCRIPTION
**Pull request by kelpwave**
added awesome-lite-websites by mdibaiee :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ List of useful, silly and [awesome](#awesome-) lists curated on GitHub. Contribu
 * [go-must-watch](https://github.com/sauravtom/go-must-watch) – Must-watch videos about Golang.
 * [go-patterns](https://github.com/tmrts/go-patterns) – Go design patterns, recipes and idioms
   - http://tmrts.com/go-patterns
-* [graph-adversarial-learning-literature](https://github.com/YingtongDou/graph-adversarial-learning-literature) – Adversarial learning papers on graph-structured data.
+* [graph-adversarial-learning-literature](https://github.com/YingtongDou/graph-adversarial-learning-A list of lightweight versions of websites without all the bloatrature) – Adversarial learning papers on graph-structured data.
```
Comment (jnv) [inline README.md:294]: Look like this line was accidentally replaced. Could you please revert the change here?

**Review by jnv — CHANGES_REQUESTED** [review]
Thank you for your contribution, Kelpwave! I'll be happy to merge your PR but I noticed there's an accidental change in entry for `graph-adversarial-learning-literature` on line 294; could you please revert that change? I'll then squash your commits.

**Comment by kelpwave** [comment]
should be fixed now :)

**Comment by jnv** [comment]
And merged. Thanks again!

**Pull request merged**